OpenSim/Simulation/Model/AppearanceMap.h

```diff
@@ -0,0 +1,100 @@
+#ifndef OPENSIM_APPEARANCE_MAP_H_
+#define OPENSIM_APPEARANCE_MAP_H_
+/* -------------------------------------------------------------------------- *
+ *                             OpenSim:  AppearanceMap.h                      *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2015 Stanford University and the Authors                *
+ * Author(s): Ayman Habib                                                     *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+
+// INCLUDE
+#include <OpenSim/Simulation/osimSimulationDLL.h>
+#include <OpenSim/Common/Property.h>
+#include <OpenSim/Common/Object.h>
+#include "Appearance.h"
+#include <iostream>
+#include <cmath>
+
+namespace OpenSim {
+
+//=============================================================================
+//=============================================================================
+/**
+ * A class that holds the map between object PathName and Appearance
+ *
```

Comment:
Who uses an appearance map? Is it a utility class to serve some other object and not intended for public use?